CHANGELOG.md

```diff
@@ -55,7 +54,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - [listMints](https://docs.x.immutable.com/reference#/operations/listMints) endpoint accepts `max_timestamp`, `min_timestamp` instead of `updated_max_timestamp`, `updated_min_timestamp` respectively
 - [BREAKING CHANGE] `x-imx-eth-address`, and `x-imx-eth-signature` headers are now required for [createOrder](https://docs.x.immutable.com/reference#/operations/createOrder),
-[cancelOrder](https://docs.x.immutable.com/reference#/operations/cancelOrder), [createTransfer](https://docs.x.immutable.com/reference#/operations/createTransfer), [createTransferV1](https://docs.x.immutable.com/reference#/operations/createTransferV1), [createWithdrawal](https://docs.x.immutable.com/reference#/operations/createWithdrawal), and [createTrade](https://docs.x.immutable.com/reference#/operations/createTrade) endpoints.
+  [cancelOrder](https://docs.x.immutable.com/reference#/operations/cancelOrder), [createTransfer](https://docs.x.immutable.com/reference#/operations/createTransfer), [createTransferV1](https://docs.x.immutable.com/reference#/operations/createTransferV1), [createWithdrawal](https://docs.x.immutable.com/reference#/operations/createWithdrawal), and [createTrade](https://docs.x.immutable.com/reference#/operations/createTrade) endpoints.
```

Comment:
is this an extra space mistake?

Reply:
It was my editor trying to figure out how to properly format it. I changed it to be in a single line for consistency.